op-node/rollup/addresses/addresses.go

```diff
@@ -0,0 +1,56 @@
+package addresses
+
+import (
+	"math/big"
+
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+type CeloAddresses struct {
+	SuggestedFeeRecipient common.Address
+}
+
+var (
+	MainnetAddresses = &CeloAddresses{
+		SuggestedFeeRecipient: common.HexToAddress("0xaa"), // TODO: replace it with the actual one
```

Comment:
Let's try to find out the correct address for this before merging. I don't see a reason why we shouldn't already know this today.

Reply:
And [we have it](https://clabsco.slack.com/archives/C0705GMH2A1/p1741702567080669): `0x7A1E98FC9a008107DbD1f430a05Ace8cf6f3FE19`